en/navigation_v2.ftl

```diff
@@ -0,0 +1,93 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+navigation-v2-mozilla = { -brand-name-mozilla }
+navigation-v2-download-firefox = Download { -brand-name-firefox }
+navigation-v2-get-a-firefox-account = Get a { -brand-name-firefox-account }
+navigation-v2-menu = Menu
+
+## Firefox menu
+
+navigation-v2-firefox-browsers = { -brand-name-firefox } Browsers
+navigation-v2-close-firefox-browsers-menu = Close { -brand-name-firefox } Browsers menu
+navigation-v2-firefox-for-desktop = { -brand-name-firefox } for Desktop
+navigation-v2-get-the-non-profit-backed = Get the non-profit-backed browser on { -brand-name-windows }, { -brand-name-mac-short } or { -brand-name-linux }.
+navigation-v2-firefox-for-android = { -brand-name-firefox } for { -brand-name-android }
+navigation-v2-get-the-customizable-mobile = Get the customizable mobile browser for { -brand-name-android } smartphones.
+navigation-v2-firefox-for-ios = { -brand-name-firefox } for { -brand-name-ios }
+navigation-v2-get-the-mobile-browser = Get the mobile browser for your { -brand-name-iphone } or { -brand-name-ipad }.
+navigation-v2-privacy-promise = Privacy Promise
+navigation-v2-learn-how-firefox-treats = Learn how { -brand-name-firefox } treats your data with respect.
+navigation-v2-firefox-blog = { -brand-name-firefox } Blog
+navigation-v2-read-about-new-firefox-features = Read about new Firefox features and ways to stay safe online.
+navigation-v2-release-notes = Release Notes
+navigation-v2-get-the-details-on-the = Get the details on the latest { -brand-name-firefox } updates.
+navigation-v2-view-all-firefox-browsers = View all { -brand-name-firefox } Browsers
```

Comment:
do we use `Firefox Browsers` as in the brand file?

Reply:
It's only singular `Firefox Browser` in brands.ftl, unless we want to add a plural form. But the way it's used here I think is correct. This "Browsers" isn't part of the brand name, it's a general term for all our browsers.

Reply:
Oh nevermind, we do have the plural. I'll update.